src/QMCApp/QMCMain.cpp

```diff
@@ -597,7 +614,7 @@ bool QMCMain::runQMC(xmlNodePtr cur, bool reuse)
     QMCDriverFactory driver_factory(myProject);
     try
     {
-      QMCDriverFactory::DriverAssemblyState das = driver_factory.readSection(cur);
+      QMCDriverFactory::DriverAssemblyState das = driver_factory.readSection(cur, estimator_manager_input_);
```

Comment:
I don't feel good estimator_manager_input_ being copied in DriverAssemblyState.
DriverAssemblyState is intended to parse `<qmc method="vmc" a="" b="">` those attributes.
estimator_manager_input_ is from main state and it should be passed driver constructor arguments just like pset, twf, ham pools.

Reply:
On reflection you are correct about this.